app/src/main/java/com/ncnf/map/NCNFMarker.java

```diff
@@ -12,20 +12,26 @@
 
 public class NCNFMarker implements ClusterItem {
 
+    public enum TYPE {
+        EVENT, ORGANIZER;
+    }
+
```

Comment:
I didn't understand very well what was that the purpose of this enum but can't you use a boolean (for example isEvent) instead of creating a new type (which could be misunderstood with the type from the event class)

Reply:
Yes you're right... I don't know why I did that now